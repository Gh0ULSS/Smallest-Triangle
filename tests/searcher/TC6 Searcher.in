-17.62, -34.92
15.09, -42.76
3.59, -13.43
-44.2, 0.74
-46.25, -6.64
-43.01, -40.93
-7.55, 32.69
-37.62, -27.68
12.74, 44.77
7.71, -10.33
47.63, -45.34
35.85, -21.04
-35.57, -38.22
-19.15, 31.61
-31.93, 8.16
13.89, -12.76
4.77, -43.72
-44.04, -29.4
18.04, -7.24
-18.59, 8.56
-4.68, -20.02
29.44, 19.9
-25.59, 7.44
2.52, 37.51
22.94, -21.21
48.02, -38.19
-8.19, 25.71
-34.8, -1.1
-46.08, 16.82
26.46, 7.3
19.53, 9.44
7.99, -4.38
34.0, 44.47
-2.59, 16.42
-43.93, 20.15
14.71, 49.31
32.19, -21.54
-11.42, 16.87
-47.74, -3.83
-44.1, 26.82
-10.91, 37.14
-41.94, -5.08
31.93, 36.4
-22.16, -8.47
-14.12, 38.42
45.77, -34.91
-32.38, -26.8
-26.67, -1.5
8.91, -23.73
-49.59, -8.11
-13.07, 6.63
45.31, 19.05
1.55, 11.76
17.62, -44.6
39.95, 28.0
37.45, 29.79
-10.76, -10.1
-39.65, 13.43
-29.12, -33.77
-15.99, -44.74
-49.98, -34.87
-39.85, -13.64
-47.45, 37.43
11.41, -35.14
-24.77, -15.26
-13.58, -37.72
34.89, 49.31
-3.4, -1.62
-15.74, -23.52
32.89, -33.86
-47.69, 45.1
2.83, -35.34
4.32, -47.3
2.81, 47.85
36.33, 19.62
-33.3, 27.19
3.26, 27.91
-17.03, -27.7
31.15, 48.49
31.83, 23.99
-27.33, 1.76
-47.21, -22.06
-24.08, 19.25
45.65, -5.28
43.7, 48.8