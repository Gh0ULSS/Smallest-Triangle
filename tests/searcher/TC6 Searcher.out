read 85 points
39.95, 28.00
37.45, 29.79
31.83, 23.99
This is a triangle